uber/templates/regform.html

```diff
@@ -367,6 +367,22 @@ <h4 class="h4-inline">
     </div>
 </div>
 
+{% if c.PAGE_PATH == '/registration/form' %}
+<div class="form-group staffing">
+    <label for="rooming" class="col-sm-2 optional-field control-label">Can Room?</label>
+    <div class="col-sm-6">
+        {% checkbox attendee.hotel_eligible %}
+        This attendee
+        {% if attendee.hotel_eligible %}
```

Comment:
I get the idea behind this, but I think it'll end up being confusing for admins.

The checkbox is always ticked if the variable is true, and not ticked if the variable is false. What will happen here is that, if an admin views an attendee's page and they are not hotel eligible, it will say "This attendee cannot request room space." next to an _unticked_ checkmark. The admin may think that the checkmark has to be ticked in order to deny room space to the attendee. This, of course, isn't true.

TLDR, the checkbox already communicates the attendee's current hotel eligibility status, so it should always say "This attendee can request room space."